### PR TITLE
Add Unit test for Breakdown Card Component

### PR DESCRIPTION
**Pull request by bipuladh**
- Unit tests for BreakDown Card Body and BreakdownCard Chart

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx [hunk 1]
```diff
@@ -11,110 +11,111 @@ import {
...
+          {...props}
+          lineHeight={1.2}
+          style={[
+            { ...datum.labels, fontSize: 9 },
+            { fill: 'black', fontSize: 8 },
+          ]}
+        />
+      </Tooltip>
+    );
+    if (datum.name === OTHER || datum.name === CLUSTERWIDE) {
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx]:
```suggestion
    if (datum.name[0] === OTHER || datum.name[0] === CLUSTERWIDE) {
```

Comment (bipuladh) [reply frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx]: https://github.com/openshift/console/pull/4852/files#diff-2b7e72b6b33459dbcc45bf9440b06f29L49
This tells a different story

File: frontend/packages/ceph-storage-plugin/src/__tests__/breakdown-chart.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  const wrapper: ShallowWrapper<LinkableLegendProps> = shallow(
+    <LinkableLegend metricModel={mockData.fakeModel} datum={legends[0]} />,
+  );
+
+  it('Renders Link', () => {
+    expect(wrapper.find(Link).exists()).toBe(true);
+  });
+
+  it('Returns tooltip', () => {
+    wrapper.setProps({ datum: { name: ['Other'], link: '#', labels: { fill: '#000' } } });
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/src/__tests__/breakdown-chart.spec.tsx]:
```suggestion
    wrapper.setProps({ datum: { name: 'Other', link: '#', labels: { fill: '#000' } } });
```

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx (hunk 1 above)
Comment (vojtechszocs) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx]: This differs from original, is that intentional?

```ts
if (datum.name[0] === OTHER || datum.name[0] === CLUSTERWIDE)
```

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx [hunk 3]
```diff
@@ -11,110 +11,111 @@ import {
...
+  },
+);
 
 export const BreakdownChart: React.FC<BreakdownChartProps> = ({
   data,
   legends,
   metricModel,
   ocsVersion,
-}) => {
-  const chartData = data.map((d: StackDataPoint, index) => (
```
Comment (vojtechszocs) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx]: The only change here is inlining `chartData` variable into JSX markup, right?

Comment (bipuladh) [reply frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx]: yes.

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx (hunk 1 above)
Comment (bipuladh) [reply frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx]: Yes. The original is erroneous.

**Pull request merged**